Documentation updates (#1512)
tidyverse/tibble: README.Rmd

tibble(x = 1:5, y = 1, z = x^2 + y)
```

> `tibble()` does much less than `data.frame()`: it never changes the type of the inputs (e.g. it keeps list columns unchanged, and never converts strings to factors!), it never changes the names of variables, it only recycles inputs of length 1, and it never creates `row.names()`. You can read more about these features in `vignette("tibble")`.

Reviewer: ```suggestion
`tibble()` does much less than `data.frame()`: it never changes the type of the inputs (e.g. it keeps list columns as is), it never changes the names of variables, it only recycles inputs of length 1, and it never creates `row.names()`. You can read more about these features in `vignette("tibble")`.
```
